omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -189,6 +242,7 @@ Should you wish to run OMERO.web automatically, a `systemd.service` file could b
     PIDFile=/opt/omero/web/omero-web/var/django.pid
     Restart=no
     RestartSec=10
+    Environment="PATH=/opt/omero/web/venv3/bin:/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin""
```

Comment:
```suggestion
    Environment="PATH=/opt/omero/web/venv3/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin"
```

Reply:
It is pointless to make change to this PR. Need to move it to the autogen part